### DISCUSSION TITLE
Can this work with youtube videos as well?

### DISCUSSION THREAD

**Discussion by darias08** [LunatiqueCoder/react-native-media-console#48] [Q&A] [answered]
Hi,

I am wondering if this could be done with youtube videos as well? I am using an api that doesn't have any storage for mp4 videos, but only youtube video ids. Please let me know if this can work with it. Thank you!

**Comment by LunatiqueCoder** [comment, marked as answer]
Hello @darias08 

Unfortunately, I believe that is not possible. Here is more information: https://github.com/criszz77/react-native-media-console/issues/41#issuecomment-1399228357

**Reply by darias08** [reply]
Got it! I was just curious if it's possible. Thanks for your reply! (:
